src/closure-defined-fs-annotation.js

```diff
@@ -1,29 +0,0 @@
-/**
- * @suppress {duplicate, undefinedVars}
- */
-var Browser;
-
-/**
- * @suppress {duplicate, undefinedVars}
- */
-var SDL;
-
-/**
- * @suppress {duplicate, undefinedVars}
- */
-var SOCKFS;
-
-/**
- * @suppress {duplicate, undefinedVars}
- */
-var Browser;
-
-/**
- * @suppress {duplicate, undefinedVars}
- */
-var __read_sockaddr;
-
-/**
- * @suppress {duplicate, undefinedVars}
- */
-var DNS;
```

Comment:
vars SDL, SOCKFS, __read_sockaddr and DNS were added originally here: https://github.com/emscripten-core/emscripten/pull/5720/files#diff-28cd435d49a673d644ba5877fc595dfcR1102 but no test. Although I doubt that PR would have been tested specifically against some external code outside the tree, so we must have had a test at that time that showed it was needed to have these vars here. (and we have not deleted tests, so I suppose these should no longer be needed)